packages/jest-preset-default/jest-preset.json

```diff
@@ -21,6 +21,11 @@
 		"**/?(*.)(spec|test).js",
 		"**/test/*.js"
 	],
+	"testPathIgnorePatterns": [
+		"/node_modules/",
+		"<rootDir>/.*/build/",
```

Comment:
Somehow I had test files copied over to build folders causing the same test fail triple times ...

I couldn't reproduce it later, however, I added ignore patterns to ensure it doesn't affect test runner.